src/types/Tags.ts

```diff
@@ -388,10 +390,10 @@ export interface TagAliases {
         rating: number,
         counter: number,
     },
-    private?: [{
+    private?: {
         ownerIdentifier: string,
-        data: string
-    }],
+        data: Buffer
```

Comment:
This is a breaking change, the correct type should be `string | Buffer` as currently the implementation accepts both `string` and `Buffer`. We should make the typing match the implementation.
If you want to change the type to `Buffer`, the implementation should be changed accordingly and the change listed in the list of breaking changes for the next release.

Reply:
I think we have to decide if we want to break the API next 0.x "major" release or split the Tags type, because as it is, it is broken anyways.

Additionally, in this case, the type has always been broken, because it returns a Buffer on read.

Reply:
I think we should do at least one release with no breaking changes with TS and then prepare the changes for the next release.